front/src/types/index.ts

```diff
@@ -0,0 +1,19 @@
+interface Todo {
+  id: string
+  title: string
+  isDone: boolean
+}
+
+interface TodoDTO {
```

Comment:
we can do `type TodoDTO = Todo;` 

Yes, by now both types the same. But declaring them as separate names - and using them separately in different cases we already separated them enough. So no need to repeat the same structure definition

Reply:
I decided to do it such way to make code more understandable. These are actually different types. It's just a happy coincidence that number of properties and their names are the same. It seems to me that in the vast majority of cases from real world we couldn't do `type TodoDTO = Todo` because there would be some difference.